addons/todo/static/src/components/todo_editable_breadcrumb_name/todo_editable_breadcrumb_name.js

```diff
@@ -0,0 +1,107 @@
+/** @odoo-module */
```

Comment:
100 LOC to adapt the size of the input to the text also looks overkill :slightly_frowning_face: